packages/block-editor/src/components/border-radius-control/README.md

```diff
@@ -0,0 +1,78 @@
+# BorderRadiusControl
+
+`BorderRadiusControl` is a React component that provides a user interface for managing border radius values. It allows users to control the border radius of each corner independently or link them together for uniform values.
+
+## Usage
+
+```jsx
+/**
+ * WordPress dependencies
+ */
+import { BorderRadiusControl } from '@wordpress/block-editor';
```

Comment:
```suggestion
import { __experimentalBorderRadiusControl as BorderRadiusControl } from '@wordpress/block-editor';
```

This component is exported with the `__experimental` prefix:

https://github.com/WordPress/gutenberg/blob/b4304f8bf6bd9b890b4108adcc326cd586a3ab4e/packages/block-editor/src/components/index.js#L37